client/beefy/src/aux_schema.rs

```diff
@@ -72,6 +73,118 @@ where
 	Ok(None)
 }
 
+mod v1 {
```

Comment:
we can drop this whole migration if we merge the PR in the same release cycle as https://github.com/paritytech/substrate/pull/13215